src/main/java/gift/product/dto/request/ProductCreateRequest.java

```diff
@@ -1,10 +1,15 @@
 package gift.product.dto.request;
 
 import gift.product.entity.Product;
+import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Size;
 
 public record ProductCreateRequest(
         Long giftId,
+        @Pattern(
+                regexp = "^[a-zA-Z0-9가-힣 ()\\[\\]+\\-&/_]*$",
```

Comment:
```suggestion
                regexp = "^[a-zA-Z0-9가-힣 ()\\[\\]+\\-&/_]+$",
```
끝의 *은 빈 문자열도 허용이니까 +로 하면 좋을 것 같아요!